tests/integration/components/self-clear-cache-test.js

```diff
@@ -0,0 +1,82 @@
+import { module, skip } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Unit | Component | self-clear-cache', (hooks) => {
+  setupRenderingTest(hooks);
+
+  skip('should show last time user cleared the cache', async function (assert) {
+    assert.expect(1);
+
+    const time = '23 March 1:23 pm IST';
+    this.set('lastTime', time);
+
+    await render(hbs`<SelfClearCache @time={{this.lastTime}} />`);
+
+    assert.equal(
+      assert.dom('[data-test-last-time]').hasText(`Last Requested : ${time}`),
+      true,
+      'Last time it was requested at 23 March 1:23 pm IST'
+    );
+  });
+
+  skip('Clear cache button should get disabled on clicking it', async function (assert) {
+    assert.expect(2);
+
+    this.set('lastTime', '23 March 1:23 pm IST');
+
+    await render(hbs`<SelfClearCache @time={{this.lastTime}} />`);
+
+    const btn = assert.dom('[data-test-clear-cache-btn]');
+    assert.equal(
+      btn.hasAttribute('disabled'),
+      false,
+      'Button is enabled right now'
+    );
+
+    await click(btn);
```

Comment:
Where do we test if the action fires for the button?

Reply:
> Where do we test if the action fires for the button?

With the click of the button, it should get disabled. That sums up that it was clicked